### ISSUE TITLE
Error compiling on Alpine linux

### ISSUE THREAD

**Issue by c4milo** [armon/statsite#191]
### Error
```
mv -f $depbase.Tpo $depbase.Po
In file included from /usr/include/limits.h:8:0,
                 from /usr/include/fortify/stdlib.h:24,
                 from src/heap.c:13:
src/heap.c:47:12: error: expected identifier or '(' before numeric constant
 static int PAGE_SIZE = 0;
            ^
src/heap.c: In function 'heap_create':
src/heap.c:103:19: error: lvalue required as left operand of assignment
         PAGE_SIZE = getpagesize();
                   ^
Makefile:526: recipe for target 'src/heap.o' failed
make[1]: *** [src/heap.o] Error 1
make[1]: Leaving directory '/statsite/statsite'
Makefile:572: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

### Dockerfile
```Dockerfile
FROM gliderlabs/alpine:3.3
WORKDIR /statsite
RUN apk add --update alpine-sdk autoconf automake linux-headers git
RUN git clone https://github.com/armon/statsite.git
RUN cd statsite && ./bootstrap.sh && ./configure && make
```

**Comment by c4milo** [comment]
Renaming PAGE_SIZE to something else made it work.

**Comment by johnkeates** [comment]
Sound like it's something the build system could theoretically fix. Is this only on Alpine?

**Comment by c4milo** [comment]
Anything using musl instead of glibc.

**Comment by armon** [comment]
I think we could probably ifdef this, @c4milo or @johnkeates  want to take a stab?

**Comment by c4milo** [comment]
on it.

**Comment by c4milo** [comment]
@armon https://github.com/armon/statsite/pull/195

**Comment by armon** [comment]
Closed by #195
